parent/pom.xml

```diff
@@ -1267,6 +1267,7 @@
       <id>parallel-tests</id>
       <properties>
         <forkCount>0.5C</forkCount>
+        <junitThreadCount>0.5C</junitThreadCount>
```

Comment:
So this will take the value 6 in our ci? I think we should set this explicitly in the scripts, just like the `forkCount`. It helps us reason about this for each specific case